src/main/scala/io/iohk/ethereum/crypto/package.scala

```diff
@@ -26,6 +26,12 @@ package object crypto {
     digest.digest
   }
 
+  def sha512(input: Array[Byte]): Array[Byte] = {
```

Comment:
@ntallar Have a look at [EC-89](https://iohk.myjetbrains.com/youtrack/issue/EC-89) shaxxx is the convention, but we can be more accurate. (that doesn't hold up merging this PR)